src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/entities/Hearing.kt

```diff
@@ -14,4 +14,5 @@ data class Hearing(
   @field:Length(max = 6)
   var agencyId: String,
   var reportNumber: Long,
+  var oicHearingId: Long,
```

Comment:
We could name this `prisonApiHearingId` or `nomisHearingId` to be more explicit?

Reply:
originally i went with nomis id, then switched it to oic_hearing as that seemed more explicit to nomis.  Happy to change it back tho